includes/admin_import.php

```diff
@@ -90,6 +91,11 @@ function tsml_import_page()
 							$meeting = array_combine($header, $meeting);
 						}
 
+						//allow theme-defined function to reformat import
+						if (function_exists('tsml_import_reformat')) {
+							$meetings = tsml_import_reformat($meetings);
+						} 
+
```

Comment:
←  To my knowledge I didn't make any code changes between lines 11 to 161 which is CSV Import code.

Reply:
I must have added it in when removing it from tsml_import_buffer_set() because we needed to run it prior to the change detection code. Obviously I didn't see the previous reformatting call on line 63, which is probably the one we should remove.